pallets/loans/src/types.rs

```diff
@@ -110,23 +113,8 @@ pub enum WriteOffAction<Rate> {
 	},
 }
 
-/// The data structure for storing loan status
-#[derive(Encode, Decode, Copy, Clone, PartialEq, TypeInfo)]
-#[cfg_attr(any(feature = "std", feature = "runtime-benchmarks"), derive(Debug))]
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub enum LoanStatus<BlockNumber> {
-	// this when asset is locked and loan nft is created.
-	Created,
-	// this is when loan is in active state. Either underwriters or oracles can move loan to this state
-	// by providing information like discount rates etc.. to loan
-	Active,
-	// loan is closed and collateral nft is transferred back to borrower and loan nft is burned
-	Closed { closed_at: BlockNumber },
```

Comment:
Because there is no longer `LoanStatus`, I would not know exactly where to store that `closed_at`. Do we really need it?

Reply:
The idea here was the following: right now, we are storing all pricing info for closed loans. This isn't really needed on chain, but it is needed for the UI. One future improvement we thought of was storing the `closed_at` and then using an archive node to get the pricing info at the block of closing, so we don't need to store it for closed loans after closing anymore.

Reply:
Sorry for side tracking but I wanted to share my thoughts regarding the "future archive node idea": Unfortunately then everyone relies on this/these single source(s) of truth without a guarantee that the provided pricing info can definitely be trusted and cannot be manipulated. On the minimum, some sort of fingerprinting technique should be applied. Pinning to IPFS with some sort of time frame to challenge the data integrity (e.g. compare data and fingerprint on full node with replication on decentralized db). could be another solution.